rtl/tests/rtl_tests.py

```diff
@@ -78,7 +78,7 @@ def test_instrumentation(event):
     "event",
     [
         pytest.param(""),
-        pytest.param("cuda:::dram__bytes,cuda:::sm__inst_executed_pipe_fma"),
+        # pytest.param("cuda:::dram__bytes,cuda:::sm__inst_executed_pipe_fma"),
```

Comment:
I am guessing this is unrelated to the PR?

Reply:
Yes, but the docker image was upgraded in the meantime